modules/srvrlss-using-cluster-logging-find-logs-services-deployed.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
+
+[id="using-cluster-logging-to-find-logs-for-services-deployed-with-Knative-Serving_{context}"]
+= Using Cluster Logging to find logs for services deployed with Knative Serving
```

Comment:
This module seems almost identical to the  modules/srvrlss-using-cluster-logging-find-knative-serving-logs.adoc module - is there a reason why we have two with the same sort of information about Kibana?

Also, can we change the module names so it's 'serverless' and not 'srvlss' please?